packages/plugin-coverage/project.json

```diff
@@ -4,27 +4,8 @@
   "sourceRoot": "packages/plugin-coverage/src",
   "projectType": "library",
   "targets": {
-    "build": {
-      "executor": "@nx/js:tsc",
-      "outputs": ["{options.outputPath}"],
-      "options": {
-        "outputPath": "dist/packages/plugin-coverage",
-        "main": "packages/plugin-coverage/src/index.ts",
-        "tsConfig": "packages/plugin-coverage/tsconfig.lib.json",
-        "additionalEntryPoints": ["packages/plugin-coverage/src/bin.ts"],
```

Comment:
I would've thought `additionalEntryPoints` had to be preserved (same for `plugin-eslint` and `plugin-js-packages`) :thinking: But E2E tests are passing, so I guess they aren't needed anymore :grin: Possibly because we switched from ESBuild bundle to TSC non-bundle.

Reply:
> Possibly because we switched from ESBuild bundle to TSC non-bundle.
Yes that is the reason! Still good catch.